packages/devtools_app/lib/main.dart

```diff
@@ -98,8 +99,8 @@ String _generateSurveyUrl() {
 
   final fromValue = uri.fragment ?? '';
 
-  // TODO(kenz): get actual value of whether user is internal or external.
-  const internalValue = 'false';
+  final ldapResponse = await HttpRequest.request('/api/getUserLdap');
```

Comment:
I would change the api from getUserLdap to getUserId
That way it can be appropriately implemented in all environments and it doesn't look like we are grabbing ldaps externally when we are not.

Reply:
I believe "all environments" is both the external pub environment and the bazel environment.

Reply:
changed to getUserId